src/scripts/main.js

```diff
@@ -1,7 +1,26 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const div = document.createElement('div');
+  const h2 = document.createElement('h2');
+  const p = document.createElement('p');
+
+  div.classList.toggle('notification');
+  div.classList.toggle(type);
+  div.scrollTop = posTop;
+  div.scrollLeft = posRight;
```

Comment:
Using `scrollTop` and `scrollLeft` to position elements is not appropriate. These properties are meant for scrolling, not for setting the position of an element. Consider using CSS styles like `top` and `right` to position the notification div.